cmd/flux/bootstrap_git.go

```diff
@@ -152,7 +152,9 @@ func bootstrapGitCmdRun(cmd *cobra.Command, args []string) error {
 		Namespace:    rootArgs.namespace,
 		TargetPath:   gitArgs.path.String(),
 		ManifestFile: sourcesecret.MakeDefaultOptions().ManifestFile,
+		SSHHostname:  repositoryURL.Host,
```

Comment:
What if we don't use ssh but only https?

Reply:
In the tests I did yesterday it seems to work with both cases, when setting the repo using https and over ssh. The problem is that, according to [this comment](https://github.com/fluxcd/flux2/pull/2216#issuecomment-993690220), the flag should be called in another way because of the behavior (it is useful to write another host in the flyx-system GitRepository, am I wrong?) so I re-used the same flag to rename it correctly in another PR